From 16e011ea995b26a5edb281dfc8b473e04b3b756a Mon Sep 17 00:00:00 2001
From: Morgane BUTSCHER <morganex.butscher@intel.com>
Date: Mon, 17 Dec 2012 16:27:49 +0200
Subject: [PORT FROM MAIN] AMTL: ttyACM and ttyPTI rights should be aligned with mts rights

BZ: 55492

ttyACM* and ttyPTI1 rights should be root radio

Change-Id: Id67448cc4427977832310eefc968767d7444ec75
Orig-Change-Id: I4fd16ec0adc4ca0e9b384959f2715cba74ba20d5
Signed-off-by: Morgane BUTSCHER <morganex.butscher@intel.com>
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
Reviewed-on: http://android.intel.com:8080/82148
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 rootdir/ueventd.rc |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 686bb33..49f6d3b 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -77,6 +77,8 @@
 /dev/usb_accessory        0660   root       usb
 /dev/tun                  0660   system     vpn
 /dev/mdmTrace             0660   root       radio
+/dev/ttyACM*              0660   root       radio
+/dev/ttyPTI1              0660   root       radio
 
 # CDMA radio interface MUX
 /dev/ts0710mux*           0640   radio      radio
-- 
1.7.4.1

